includes/OAuth/OAuthAuthenticator.php

```diff
@@ -59,35 +63,22 @@ public function setAuthenticationEndpoint(string $endpoint)
      * @throws \Exception With message.
      * @return string
      */
-    public function authenticate(string $clientId, string $clientSecret): object
+    public function getToken(string $clientId, string $clientSecret): OAuthToken
```

Comment:
I like authenticate better as a name for this function. Since it authenticates using the given credentials.